### DISCUSSION TITLE
Ability to configure query timeout for PostgreSQL Datasource

### DISCUSSION THREAD

**Discussion by serhii-bielozorov** [grafana/grafana#69228] [Data Sources]
Hello team,

It would be great to have some query_timeout configuration option for PostgreSQL Datasource. Seems like the default timeout is 1 minute which is non-configurable and is not enough for heavy-weight queries.

Thanks.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82631. Feel free to continue the discussion around this there. Thank you!
